### PR TITLE
Removed deploymentconfig frontend autoscaled from the Autoscaling for…

### PR DESCRIPTION
**Pull request by ghost**
… CPU Utilization example

@ghyde Please review. This is the follow-up to https://github.com/openshift/openshift-docs/pull/6081.

Also, does this change apply to OCP 3.2 - 3.9, online, and dedicated?

### CODE REVIEW

**Comment by ghyde** [comment]
The docs for OCP 3.2-3.5 looks like this:
```
$ oc autoscale dc/frontend --min 1 --max 10 --cpu-percent=80
deploymentconfig "frontend" autoscaled
```

Docs for 3.6-3.7, online, and dedicated looks like this:
```
$ oc autoscale dc/frontend --min 1 --max 10 /
  --cpu-percent=80 deploymentconfig "frontend" autoscaled
```

The text "deploymentconfig 'frontend' autoscaled" is the output from the command. So, I think the better option will be to move that text to a new line instead of removing it.

**Comment by ghyde** [comment]
I updated my pull request (#6081) to reflect this correction.

**Comment by tmorriso-rh** [comment]
@ghyde Thanks, I updated with your changes. Please review.

**Comment by ghyde** [comment]
@tmorriso-rh LGTM

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL. No rev history needed.

File: dev_guide/pod_autoscaling.adoc [hunk 1]
```diff
@@ -92,8 +92,8 @@ are given default values from the {product-title} server.
 For example:
 
 ----
-$ oc autoscale dc/frontend --min 1 --max 10 /
-  --cpu-percent=80 deploymentconfig "frontend" autoscaled
+$ oc autoscale dc/frontend --min 1 --max 10 / --cpu-percent=80 
+deploymentconfig "frontend" autoscaled
```
Comment (mburke5678) [inline dev_guide/pod_autoscaling.adoc]: @tmorriso-rh @ghyde Do we need the `/` between `--max 10` `--cpu-percent=80`?

Comment (ghyde) [reply dev_guide/pod_autoscaling.adoc]: No. My PR (#6081) doesn't have that.

**Comment by tmorriso-rh** [comment]
Issue where this PR was automatically closed when trying to rebase. Opening a new PR to make this change.

**Comment by vikram-redhat** [comment]
@tmorriso-rh - make sure to remove milestones and labels from PRs that are closed without merging.

**Pull request closed without merging**